Content.Server/Chat/ChatSystem.cs

```diff
@@ -329,19 +334,45 @@ private IEnumerable<INetChannel> GetDeadChatClients()
             .Select(p => p.ConnectedClient);
     }
 
-    private string SanitizeMessageCapital(EntityUid source, string message)
+    private (string, int) RadioPrefix(EntityUid source, string message)
```

Comment:
This should be returning the prototype string, not the channel int. Also comments.